kubernetes/argocd/KustomizeSpec/default.dhall

```diff
@@ -0,0 +1,4 @@
+{ commonLabels = None { additionalProperties : Text }
+, images = [] : List Text
```

Comment:
In kubernetes empty list and `null` often mean different things. Can we make this an optional list for consistency? I plan to switch to use `--omitNull` soon in our plugin (see https://github.com/dhall-lang/dhall-kubernetes/issues/86)